parachains/integration-tests/statemine/xcm/1_dmp.yml

```diff
@@ -69,16 +69,16 @@ tests:
                   ]
                   events:
                     - name: xcmPallet.Attempted
-                      attribute:
-                        type: XcmV2TraitsOutcome
-                        isComplete: true
+                      attributes:
+                        - type: XcmV2TraitsOutcome
+                          xcmOutcome: Complete
                     - name: dmpQueue.ExecutedDownward
                       chain: *assets_parachain
-                      attribute:
-                        type: XcmV2TraitsOutcome
-                        isComplete: true
-                        threshold: [10, 10]
-                        value: 1,021,973,000
+                      attributes:
+                        - type: XcmV2TraitsOutcome
```

Comment:
v3 is set on init, but here v2.
check in the output if the assertions did really happen

Reply:
just got 0 passing. Have changed this locally but not got 1_dmp passing yet. Will upload my changes.